doc/how-to/initialise.md

```diff
@@ -84,8 +90,10 @@ Complete the following steps to initialise MicroCloud:
       Encrypting a disk will store the encryption keys in the Ceph key ring inside the Ceph configuration folder.
 
       ```{note}
-      Encryption requires a kernel with `dm_crypt` enabled.
-      See {doc}`how full disk encryption works <microceph:explanation/full-disk-encryption>` in the MicroCeph documentation for more information.
+      Disks to be encrypted require a kernel with `dm_crypt` enabled. See {doc}`how full disk encryption works <microceph:explanation/full-disk-encryption>` in the MicroCeph documentation for more information.
+
+      If you have not enabled `dm_crypt` on a cluster member disk that you want to encrypt, open a separate terminal and enable it before you continue. If the disk is on the cluster member on which you ran `microcloud init`, stop the initialisation process and start it again after enabling `dm_crypt`.
```

Comment:
Isn't `dm_crypt` enabled per cluster member and not disk? 

Even if it's the same cluster member you can keep staying in the init process and launch another terminal to enable `dm_crypt`. You don't have to restart the init process.
The only requirement is that you are still in the questionnaire and MicroCloud hasn't yet started to form and configure the clusters.

Reply:
I just noticed I need to fix `dm_crypt `to `dm-crypt` as well

Reply:
Fixed all above concerns